site/src/pages/index.astro

```diff
@@ -62,7 +62,9 @@ function MyApp() {
       <li>
         Listeners for OpenLayers events can be added with
         <code>on</code> -prefixed props. For example, <code>onChange</code> will
-        be called for <code>change</code> events.
+        be called for <code>change</code> events. Colons in props can cause issues
+        with React, so we also expose props like <code>onChange_center</code> along 
```

Comment:
Is this a problem with React or a problem with Babel?

Reply:
Unclear if it's Babel or React.

Quoting https://babeljs.io/docs/babel-preset-react#throwifnamespace :

> Though the JSX spec allows this, it is disabled by default since React's JSX does not currently have support for it.

I think some versions of react will just not take into account props with colons in the name. To prevent these silent runtime errors, babel explicitly fail at build time by default. Other bundlers/preset might not fail at build time, but still leads to wrong run time behavior.